config/m3aggregator.yaml

```diff
@@ -57,6 +57,16 @@ msgpack:
     iteratorPool:
       size: 4096
 
+runtimeOptions:
+  kvConfig:
+    namespace: /m3aggregator
+    environment: production
+  writeValuesPerMetricLimitPerSecondKey: write-values-per-metric-limit-per-second
+  writeValuesPerMetricLimitPerSecond: 0
+  writeNewMetricLimitClusterPerSecondKey: write-new-metric-limit-per-second
```

Comment:
nit: can this be new-metric(-limit)-per-shard-per-second?

Reply:
This is the cluster-wide limit per second, not the per-shard limit. Otherwise whoever sets this needs to do math to figure out what the proper limit is for the entire cluster.